CHANGELOG.md

```diff
@@ -14,6 +14,7 @@
 - **depends:** Check if extractor is available ([#4042](https://github.com/ScoopInstaller/Scoop/issues/4042))
 - **decompress:** Fix nested Zstd archive extraction ([#4608](https://github.com/ScoopInstaller/Scoop/issues/4608))
 - **shim:** Fix PS1 shim error when in different drive in PS7 ([#4614](https://github.com/ScoopInstaller/Scoop/issues/4614))
+- **shim:** Fix shell shim error when in WSL1(Ubuntu 20.04) ([#4637](https://github.com/ScoopInstaller/Scoop/issues/4637))
```

Comment:
```suggestion
- **shim:** Fix `sh` shim error in WSL ([#4637](https://github.com/ScoopInstaller/Scoop/issues/4637))
```

I think it would affect any WSL distro, so no need to say Ubuntu